recipe/meta.yaml

```diff
@@ -39,6 +37,11 @@ test:
     - pydeck.io
     - pydeck.nbextension
     - pydeck.widget
+  requires:
+    - pip
+    - ipython_genutils
```

Comment:
Why does it require ipython_genutils?

Reply:
import pydeck.widget

Reply:
Our ipywidgets 7.6.5 is missing a dependency on ipython_genutils.

Reply:
what would you propose, fix ipywidgets first ?

Reply:
yes, I opened a PR for that: https://github.com/AnacondaRecipes/ipywidgets-feedstock/pull/9 if you can approve.